src/main/resources/templates/user.html

```diff
@@ -21,12 +21,12 @@
 
     <div class="row">
 
-        <div class="col-md-4" th:each="user : ${users}">
+     <!--   <div class="col-md-4" th:each="user : ${users}">
             <h2 class="h2" th:text="'Witaj,' + ${user.firstName} "> </h2>
             <h3 class="h2" th:unless="${user.admin}" th:text="'Witaj, '+${user.firstName} +' '+ ${user.lastName} "></h3>
```

Comment:
Własnie nie wiem czy zostawiamy komentarze czy po prostu nie usuwamy :)

Reply:
to w sumie był test tylko pull requesta,  a komentarze bym wyjebał wszystkie tak btw :)

Reply:
to ten request do smieci raczej co nie?